common/beerocks/hostapd/configuration.cpp

```diff
@@ -182,6 +185,17 @@ std::string Configuration::get_vap_value(const std::string &vap, const std::stri
     return it_str->substr(key_eq.length());
 }
 
+void Configuration::disable_all_ap_vaps() {
+    for_each(
+            m_hostapd_config_vaps.begin(),
+            m_hostapd_config_vaps.end(),
+            [this](const std::pair<std::string, std::vector<std::string>> &vap) {
+                if ( get_vap_value(vap.first, "mode") == "ap" ) {
+                    set_create_vap_value(vap.first, "start_disabled", "1");
+                }
+            });
+}
```

Comment:
This function is never going to be used. There is no bwl function to disable all VAPs, and there's no need for it.